src/components/molecules/UserProfileHeader.tsx

```diff
@@ -21,17 +21,17 @@ const useStyles = makeStyles({
 });
 
 interface IUserProfileHeader {
-  name: string;
+  firstName?: string;
 }
-const UserProfileHeader: React.FC<IUserProfileHeader> = ({ name }) => {
+const UserProfileHeader: React.FC<IUserProfileHeader> = ({ firstName }) => {
   const classes = useStyles();
 
   return (
     <div className={classes.profile}>
       <a className={classes.link} href={LINK}>
         <Typography.Body1>LOGOUT</Typography.Body1>
       </a>
-      <Typography.Body1>{`שלום ${name}`}</Typography.Body1>
+      <Typography.Body1>{` שלום${firstName}`}</Typography.Body1>
```

Comment:
space?
do we still want to show 'שלום' even there is no name?
{ firstName = '' }

Reply:
i18n!!!

Reply:
done
:)